data-prepper-plugins/service-map-stateful/build.gradle

```diff
@@ -19,7 +19,7 @@ dependencies {
     implementation 'commons-codec:commons-codec:1.15'
     implementation 'io.micrometer:micrometer-core'
     implementation 'com.fasterxml.jackson.core:jackson-databind'
-    implementation("io.opentelemetry:opentelemetry-proto:${versionMap.opentelemetryProto}") {
+    implementation("io.opentelemetry.io:opentelemetry-proto:${versionMap.opentelemetryProto}") {
```

Comment:
@kmssap , I think this should be `io.opentelemetry.proto`. That may be why the build is failing.

Reply:
Thanks, this has been fixed in the latest commit.

Reply:
Thanks, this has been fixed with the latest commit.